tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1011,6 +1011,46 @@ def _get_python_version(self):
         return "python{}.{}".format(sys.version_info.major, sys.version_info.minor)
 
 
+@skipIf(
+    ((IS_WINDOWS and RUNNING_ON_CI) and not CI_OVERRIDE),
+    "Skip build tests on windows when running in CI unless overridden",
+)
+class TestBuildCommand_ExcludeResources(BuildIntegBase):
+    template = "many-more-functions-template.yaml"
+
+    @parameterized.expand(
+        [
+            ((), None),
+            ("FunctionOne", None),
+            ("FunctionThree", None),
+            ("FunctionOne", "FunctionOne"),
+            ("FunctionOne", "FunctionTwo"),
+        ]
+    )
+    @pytest.mark.flaky(reruns=3)
+    def test_build_without_resources(self, excluded_resources, function_identifier):
```

Comment:
is `excluded_resources` a string or a list/tuple? The input params are strings but then is treated like a list in `_verify_resources_excluded`?

Also, since `--exclude` can accept multiple values, should there be an input that specifies multiple resources to exclude?

Reply:
Right, these should be tuples. I will fix this in the next commit.